packages/cli/src/oclif/ZapierBaseCommand.js

```diff
@@ -27,7 +27,11 @@ class ZapierBaseCommand extends Command {
     this.debug('flags are', this.flags);
     this.debug('------------');
 
-    this.throwForInvalidAppInstall();
+    // only a few commands need the core package to run
+    // so we watch for them here and then check for zapier-platform-core
+    if (this.requiresCore()) {
+      this.throwForInvalidAppInstall();
+    }
```

Comment:
I _think_ I set the requirement too `true` for all the packages that do require core but I'm no where near 100% sure on that. And I don't have a huge amount of faith that the unit tests would actually catch it if it's wrong for a command.